components/SingleSelectRadio/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import { ComponentContainer, RadioInput, RadioLabel } from './styles';
+
+interface Option {
+  label: string;
+  value: string;
+}
+
+interface SingleSelectRadioGroupProps {
+  name: string;
+  options: Option[];
+  defaultValue?: string;
+  onChange?: (value: string) => void;
+}
+
+export default function SingleSelectRadioGroup({
+  name,
+  options,
+  defaultValue,
+  onChange,
+}: SingleSelectRadioGroupProps) {
+  const [selectedValue, setSelectedValue] = useState(
+    defaultValue || options[0].value,
```

Comment:
this is a good idea! however, we don't want any option to be selected by default in onboarding, so it should be possible for selectedValue to be null/undefined. 

I think we should change this to: 

`const [selectedValue, setSelectedValue] = useState<string | null>(defaultValue || null);`